cmd/generate.go

```diff
@@ -81,6 +82,40 @@ func genProcessCluster(cmd *cobra.Command, clusterName string, p *ants.Pool) {
 		}
 	}
 
+	// list of components in cluster
+	for c, _ := range clusterComponents {
+		var clComps []string
```

Comment:
what does clComps mean?

Reply:
I think you should rename this to something more meaningful; especially if it means "clusterComponents"